### ISSUE TITLE
Can this tweak affect gtk4?

### ISSUE THREAD

**Issue by mthw0** [PCMan/gtk3-nocsd#39]
I am asking because it might cause a crash of gtk4-demo. See: https://gitlab.gnome.org/GNOME/gtk/issues/1251

**Comment by mthw0** [comment]
Answer to my question: It looks like it does. Could you fix it?

**Comment by rilian-la-te** [comment]
I think it will affect all GTK versions, because it is a LD_PRELOAD hack. It just overrides all functions with same signature.

**Comment by luigifab** [comment]
Yes it affect GTK 4, and it almost works with [gtk4-widget-factory 3.98.2-1](https://packages.debian.org/experimental/gtk-4-examples).
But, the minimize/maximize/close buttons are not removed.

![gtk4](https://user-images.githubusercontent.com/31816829/79135975-7b9fae80-7db0-11ea-8885-2349ef6227ea.png)

**Comment by MorsMortium** [comment]
https://codeberg.org/MorsMortium/GTK4-NoCSD
